src/js/components/Chart.js

```diff
@@ -582,6 +582,25 @@ export default class Chart extends Component {
     return (pos1 && pos2 && pos1.endX > pos2.startX && pos1.startX < pos2.endX);
   }
 
+  _splitLabel (label, position, labelY) {
```

Comment:
I would see this being done outside the chart as there are some assumptions in the component label. For example, we are assuming that a comma (new delimiter prop) will be present so that we can create tspans to separate it.

I wonder if you can create a label string with a `\n` in it when invoking the Chart component.